core/src/epicli/data/common/ansible/playbooks/roles/postgresql/tasks/upgrade/extensions/replication/repmgr-for-pg-10-main.yml

```diff
@@ -0,0 +1,42 @@
+---
+- name: repmgr for PG 10 | Load default variables for upgrade
```

Comment:
Do we need version value in task names? May be `repmgr for PG {{ pg_old_version }}...`

Reply:
Not sure if we need it but we may decide later - during integration of repmgr upgrade with PostgreSQL upgrade. For now I applied the suggestion with variable.